packages/boot/test/bootstrapTests/lca.test.ts

```diff
@@ -89,4 +89,30 @@ test.serial('stakeBld', async t => {
       },
     },
   });
+  t.like(wd.getLatestUpdateRecord(), {
+    status: { id: 'request-delegate', numWantsSatisfied: 1 },
+  });
+
+  await t.throwsAsync(
+    wd.executeOffer({
+      id: 'request-delegate-504',
+      invitationSpec: {
+        source: 'continuing',
+        previousOffer: 'request-stake',
+        invitationMakerName: 'Delegate',
+        invitationArgs: ['agoric1validator1', { brand: BLD, value: 504n }],
+      },
+      proposal: {
+        give: {
+          // @ts-expect-error XXX BoardRemote
+          In: { brand: BLD, value: 504n },
+        },
+      },
+    }),
+    {
+      // FIXME should receive "simulated packet timeout"
+      message:
+        /syscall.callNow failed: device.invoke failed, see logs for details/,
+    },
```

Comment:
This newly added test, despite the broken error propagation through runUtils, passes on current `master`

Reply:
Temporarily fixed via [b59ccfe](https://github.com/Agoric/agoric-sdk/pull/9591/commits/b59ccfe5c4f77051f9a1eeb864775d188c835d47)